src/components/chat/chat-window.tsx

```
@@ -0,0 +1,220 @@
+
+interface Message {
+  id: string;
+  content: string;
+  senderId: string;
+  roomId: string;
+  createdAt: Date;
+}
+
+export function ChatWindow({ roomId }: ChatWindowProps) {
+  const [message, setMessage] = useState("");
+  const [messages, setMessages] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)